### ISSUE TITLE
missing pose_hrnet_w48_256x256.pth for mpii

### ISSUE THREAD

**Issue by fenaux** [leoxiaobin/deep-high-resolution-net.pytorch#223]
Point 7 in the front page mention pose_hrnet_w48_256x256.pth is available for mpii dataset. but I could not find it neither on google drive nor on one drive

**Comment by smj007** [comment]
Yeah the weights file for this variant is not available, please do add it in the drive links

**Comment by smj007** [comment]
I guess we just have to use the weights and config for the MPII w32 version again as their results are nearly the same
